database/migrations/2023_07_13_182809_dictionaries_and_positions.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+          Schema::create('dictionaries', function (Blueprint $table) {
+              $table->char('id', 36)->collation('ascii_bin')->primary();
+              $table->char('facility_id', 36)->collation('ascii_bin')->nullable();
+              $table->string('name');
+              $table->boolean('is_fixed');
```

Comment:
do czego ma służyć te pole?

Reply:
którego albo nie może zmienić/usunąć nawet admin, bo to są rzeczy odwołujące się do kodu albo generalnie niezmienne

ale nie aż tak niezmienne jak Enum, bo można dodawać pozycje

Reply:
a to nie lepiej by jakoś nazwać is_systemic albo coś w tym stylu? skąd taka konwencja?

Reply:
I w ogóle dopisz w komentarzu te mądrości. Uwaga ogólna do właściwie wszystkich miejsc gdzie reviewer pyta "a co to jest", "a czemu to tak jest" - pyta bo najwyraźniej brakuje komentarza w kodzie.

Reply:
W ogóle nie jestem pewien czy to rozgraniczenie na pewno jest sensowne. Bo dni tygodnia faktycznie raczej nikt nie pozmienia, ale na przykład płcie to jest enum czy słownik? Można by uznać że "męska", "żeńska" i "inna" na sto procent pokrywają wszystkie możliwości, ale jeżeli z jakiegoś powodu ktoś zechce odnotowywać dodatkowe informacje, i z powodów np. ideologicznych będzie bardzo chciał to robić w polu płeć, to będzie to można rozszerzyć czy absolutnie nie?

Reply:
Idea jest taka, że teoretycznie dowolna placówka może dodawać pozycje do tego słownika. Czy to rzeczywiście tak będzie, że is_fixed na słowniku oznacza że nie można go zmienić, ale można dodawać pozycje, to się pewnie jeszcze okaże. No może z tym "nieznana" to się pospieszyłem, że to może być pozycja tych placówek które działają przed urodzeniem dziecka.

No nawet sobie myślałem, że może rzeczywiście warto zrobić taki jeden .md na tabelę z listą kolumn i kawałkiem słownego opisu. Inna sprawa, biznesowo istotne jest api a nie struktura bazy. Ale zadanie mogę sobie dodać na to